jdk8/server.xml

```diff
@@ -89,9 +89,10 @@
     -->
 
     <!-- Define an AJP 1.3 Connector on port 8009 -->
+    <!--
```

Comment:
Let's just pray this doesn't mess with the whole ecosystem, because it makes no sense that we set the maxSwallowSize but don't actually need that port, right?